sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestVariousOddities.java

```diff
@@ -121,4 +122,30 @@ public void testNullQueryReturn() {
      * This interface should not be loaded by any test other than {@link TestVariousOddities#testConcurrentHashCode()}.
      */
     public interface SpiffyConcurrent extends SqlObject {}
+
+    @Test
+    public void testInterfaceMIAbiguity() {
+        assertThatThrownBy(() ->
+                dbRule.getSharedHandle().attach(SpiffyParallelUniverse.class))
+            .isInstanceOf(UnableToCreateSqlObjectException.class)
+            .hasMessageContaining("SpiffyParallelUniverse has ambiguous methods");
+    }
+
+    @Test
+    public void testInterfaceMIResolution() {
+        dbRule.getSharedHandle().attach(SpiffyUniverseSaved.class);
+    }
+
+    public interface SpiffyWithMustache {
+        @SqlQuery("select id, name from something where id = :id")
+        Something byId(@Bind("id") long id);
+    }
+
+    public interface SpiffyParallelUniverse extends Spiffy, SpiffyWithMustache {}
```

Comment:
can we get less arbitrary names here? `ParentFoo extends SubFoo1, SubFoo2` or something

Reply:
Is the joke not funny at all? :(  Coming up with silly test names is one of the only fun parts of writing tests, and I was trying to re-use what was already there (the whole `Spiffy`) thing.  I guess I can make a separate hierarchy.

Reply:
I laughed

Reply:
Not much of a hierarchy, just 1 interface extending two others and that's it... It's not like we need to use the made-up domain here